products/android.md

```diff
@@ -39,7 +39,7 @@ releases:
 
 -   releaseCycle: "10"
     codename: Queen Cake
-    eol: false
+    eol: true
```

Comment:
As per the last security bulletin with an update for 10: https://source.android.com/docs/security/bulletin/2023-02-01

https://source.android.com/docs/security/bulletin/2023-03-01 doesn't cover 10.

```suggestion
    eol: 2023-02-08
```

Reply:
Apologies, typo, should be 06/03/2023

Reply:
@marcwrobel thoughts on how we can decide this?

Reply:
For "whether or not to consider an Android version EOL" I think the strategy used here is the good one : _when a version is not listed anymore in the security bulletin this means it is EOL_. This is consistent with what I observed on https://source.android.com/docs/security/bulletin/2022 (Android 10 is always listed in the _YYYY-MM-DD security patch level vulnerability details_ section). Note that this seem to be the strategy used by Wikipedia contributors (on https://en.wikipedia.org/wiki/Android_10).

I think using the "last security patch date" as the EOL date is the best choice, considering there is only one patch per month (or so it seems ?). And I think the exact day in this case should be the first day of the month. I looked through a dozen of security bulletins and the title is always _YYYY-MM-01 security patch level vulnerability details_. But ultimately I don't think it is very important.

Thanks @YandexStudio for sharing the link to the security bulletins (I did not found it when I searched). With this link I found https://android.googlesource.com/platform/build/+refs, which could be used to automate the update of new `latest` / `latestReleaseDate` fields.

Reply:
I work as a Cyber Essentials (CE) assessor and we need to check compliancy on operating systems and we go by the Android security bulletin to check for support.  If the OS is not listed as part of it and therefore not receiving security updates it is considered unsupported.  I have this backed up from the org that runs CE, in which they use the same link to check this info.  

This info is released the first Monday of every month, unless it is a public holiday in which it will be released the next working day.  The March edition was released on 06/03/2023

Reply:
Thanks for the confirmation. Going to set it to 2023-03-06 for Android 10 + Merge, and will file an issue to fill the rest.